src/libs/ReportActionsUtils.ts

```diff
@@ -214,15 +214,53 @@ function getSortedReportActions(reportActions: ReportAction[] | null, shouldSort
         return (first.reportActionID < second.reportActionID ? -1 : 1) * invertedMultiplier;
     });
 
-    // If shouldMarkTheFirstItemAsNewest is true, label the first reportAction as isNewestReportAction
-    if (shouldMarkTheFirstItemAsNewest && sortedActions?.length > 0) {
-        sortedActions[0] = {
-            ...sortedActions[0],
-            isNewestReportAction: true,
-        };
+    return sortedActions;
+}
+
+/**
+ * Returns the range of report actions from the given array which include current id
+ * the range is consistent
+ *
+ * param {ReportAction[]} array
+ * param {String} id
+ * returns {ReportAction}
+ */
+function getRangeFromArrayByID(array: ReportAction[], id?: string): ReportAction[] {
+    let index;
+
+    if (id) {
+        index = array.findIndex((obj) => obj.reportActionID === id);
+    } else {
+        index = array.findIndex((obj) => obj.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD);
     }
 
-    return sortedActions;
+    if (index === -1) {
+        return [];
```

Comment:
Let's add a `Log.hmmm` here?

Reply:
oh, nevermind then

Reply:
So I assume this happens because the `sortedReportActions` often start out being an empty array before they're loaded from `Onyx`? Sorry for the confusion, let's remove the `Log.hmmm` again.

Reply:
Okay, perhaps it does make sense since I checked if the length is more than 0 before I called it. 
So, this can be included, but at the same time, it might not be called. I've added it here as a safety measure